### PR TITLE
Bug 1877906: remove unwanted flag "--setopt=tsflags=nodocs"

### PR DESCRIPTION
**Pull request by Rajpratik71**
From many discussions and references , redhat based system container
images are built using yum's nodocs option, which helps reduce the size
of the image.

The discussions , references and best practices can be found at

https://github.com/CentOS/sig-cloud-instance-images/issues/21

http://docs.projectatomic.io/container-best-practices/

https://github.com/jumanjiman/docker-tsflags/blob/master/Dockerfile

https://bugzilla.redhat.com/show_bug.cgi?id=1512663

Signed-off-by: Pratik Raj <rajpratik71@gmail.com>

### CODE REVIEW

**Comment by periklis** [comment]
@Rajpratik71 I don't quite get the point with this PR, why should we want the docs in our containers? Do you miss anything else beyond docs?

**Comment by Rajpratik71** [comment]
> @Rajpratik71 I don't quite get the point with this PR, why should we want the docs in our containers? Do you miss anything else beyond docs?

# Centos and Fedora base images have an /etc/yum.conf with:
#   tsflags=nodocs
#
# That option causes rpm to skip files in  /usr/share/doc
# and thereby make the built image smaller.

Also , mentioned in conversation at [Redhat Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1512663)

So , removing unwanted flag  "--setopt=tsflags=nodocs".

**Comment by periklis** [comment]
> > @Rajpratik71 I don't quite get the point with this PR, why should we want the docs in our containers? Do you miss anything else beyond docs?
> 
> ### Centos and Fedora base images have an /etc/yum.conf with:
> ### tsflags=nodocs
> ### That option causes rpm to skip files in /usr/share/doc
> ### and thereby make the built image smaller.
> 
> Also , mentioned in conversation at [Redhat Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=1512663)
> 
> So , removing unwanted flag "--setopt=tsflags=nodocs".

:facepalm: I should read more carefully.

/ok-to-test

**Comment by periklis** [comment]
@Rajpratik71 Thank you very much for this small improvement. We really appreciate this. However, openshift release cycle is on the middle of feature-freeze, thus everything super relevant needs a Bugzilla issue. I recommend to keep this PR as is for now w/o opening a Bugzilla issue because it does not severely affect the release in the works. The need for `bugzilla/valid-bug` label will be removed in 2 weeks from now. After getting an approval - which we will certainly get for this - the bot will take care merging this PR.

**Review by periklis — APPROVED** [review]
/lgtm

**Comment by Rajpratik71** [comment]
@periklis  and @jcantrill  , please get approval from @alanconway  and merge it or atleast stop the freaking bot  it building every hour.

**Review by alanconway — APPROVED** [review]
/approve

**Review by periklis — APPROVED** [review]
/lgtm

**Pull request merged**
